qiskit/pulse/commands/snapshot.py

```diff
@@ -68,5 +82,9 @@ def to_instruction(self):
     # pylint: enable=arguments-differ
 
     def __repr__(self):
-        return '%s(%s, %s) -> %s' % (self.__class__.__name__, self.name,
-                                     self.type, self.channels)
+        if self.label == self.name:
```

Comment:
Maybe we don't need this check and can always print out the name to be more consistent?

Reply:
Sure. Changed it to always print the name.